repos/fdbt-site/src/utils/index.ts

```diff
@@ -261,3 +263,33 @@ export const objectKeyMatchesExportNameExactly = (objectKey: string, exportName:
 export const fareTypeIsAllowedToAddACap = (fareType: string): boolean => {
     return !!fareType && ['single', 'return', 'flatFare'].includes(fareType);
 };
+
+export const getUniqueServices = (services: ServiceType[]): ServiceType[] => {
```

Comment:
Can we not just have one function which does this, but it just takes this as an argument:

`{ lineId: string, startDate: string, endDate: string | undefined } []`

And then you can just pass the MyFaresService[] and the ServiceType[] to the function, and it wont care that they have additional information?

Reply:
what about the return from the function